tests/README.md

```diff
@@ -7,6 +7,13 @@ This directory contains, primarily, e2e tests for the Oasis Indexer. They have b
 
 ## Setup for E2E Tests
 
-To ensure that tests behave as expected, you should have either the local environment or Docker environment configured as per the [top-level README](../README.md#docker-development).
+To ensure that tests behave as expected, you should have either Docker environment configured as per the [top-level README](../README.md#docker-development).
```

Comment:
It would be nice to encourage a set of containers (I guess just the db, at this point) that's separate for e2e, as opposed to regular development. The e2e set always needs to start clean and can be killed right after the tests.

We could package the whole operation (build fresh docker image, bring up docker compose, exec into the indexer and run tests, tear down docker compose) into a single Makefile operation.

Reply:
I ack this nice to have.

Reply:
https://github.com/oasisprotocol/oasis-indexer/issues/202